README.md

```diff
@@ -369,4 +370,255 @@ You may find it helpful to see the Kubernetes resources visually which can be ac
+(depending on the size of RAM in your workstation). 
+
+### 4. Change/Verify Context
+
+If you have previously deployed the DBP to AWS or minikube you will need to change/verify that the `docker-for-desktop` context is being used.
+
+```bash
+kubectl config current-context                 # Display the current context
+kubectl config use-context docker-for-desktop  # Set the default context if needed
+```
+
+### 5. Restart Docker  
+
+Docker can be faulty on its first start. So, it is always safer to restart it before proceeding. Right click on the Docker icon in the system tray, then left click "restart...". 
+
+### 6. Install Helm
+
+Install chocolatey using command prompt (with admin rights). 
+
+```bash
+@"%SystemRoot%\System32\WindowsPowerShell\v1.0\powershell.exe" -NoProfile -InputFormat None -ExecutionPolicy Bypass -Command "iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))" && SET "PATH=%PATH%;%ALLUSERSPROFILE%\chocolatey\bin"
+```
+
+Install helm
+
+```bash
+choco install kubernetes-helm
+```
+
+Initialize Tiller (Server Component)
+
+```bash
+helm init
+```
+
+### 7. Create your namespace
+
+```bash
+kubectl create namespace $DESIREDNAMESPACE
+```
+
+### 8. Pull secrets
+
+Log in to quay.io
+```bash
+docker login quay.io
+```
+Give username and password if prompted.
+
+In windows explorer, go to the current location of your command line.
+Create the file secrets.yaml, and past the following into it. 
+
+```bash
+apiVersion: v1
+kind: Secret
+metadata:
+  name: quay-registry-secret
+type: kubernetes.io/dockerconfigjson
+data:
+  .dockerconfigjson: <replaceThis>
+```
+Back in your command line, generate a base64 value for your dockercfg, this will allow Kubernetes to access quay.io
+
+```bash
+certutil -encode "%USERPROFILE%\.docker\config.json" tmp.b64 && findstr /v /c:- tmp.b64
+```
+From the output, copy the string located under:
+
+```bash
+CertUtil: -encode command completed successfully.
+```
+
+In the secrets.yaml file, replace <replaceThis> with the copied string. 
+
+Note that when you paste the string in the data section, it may be pasted with new lines in it. So, make sure to take out the new lines. And leave a single space between "data:" and the string. 
+
+Now in your command line, remove the file that was just created as it is no longer needed. 
+
+```bash
+del tmp.b64
+```
+
+Create the secret in your namespace. 
+
+```bash
+kubectl create -f secrets.yaml --namespace $DESIREDNAMESPACE
+```
+
+### 9. Download the alfresco helm chart
+
+```bash
+helm repo add alfresco-incubator https://kubernetes-charts.alfresco.com/incubator
+```
+
+### 10. Authorize connections
+
+Go to the config.json file in "C:\Users\<YourUserName>\.docker", and check that there is a string after "auth", such as in the following example.
+
+```bash
+"auth": "klsdjfsdkifdsiEWRFJDOFfslakfdjsidjfdslfjds"
+```
+
+Do the following command to find your ipv4 address and copy it to your clipboard.
+
+```bash
```

Comment:
we might as well suggest using the following command in PS:
```invoke-RestMethod http://ipinfo.io/json | Select -exp ip```

Reply:
@ehardon Do we need the public IP? Or would we just need the IP of the machine we are connecting to? ie if I were to set this up at home and used the public IP I would need to also open ports etc. on my router.

Reply:
@jottley based on Ray's steps for deploying on Mac, I saw he was fetching the ip of the machine using "`ipconfig getifaddr en0` so I tried finding something similar for windows and that was the best option I found, but you might be right as I haven't tried it at home.

Reply:
But @ehardon those are 2 different cases, `ipconfig getifaddr en0` just returns an ip address of a network interface en0, regardless of whether it's externally visible or not. There's a good chance an external IP address (as returned by `ipinfo.io`) will not be accessible and for this purpose we don't need to route this traffic outside the local network interface anyway.

Reply:
I see... my bad then!